docs/contributing.rst

```diff
@@ -86,7 +86,7 @@ Change directory to the ``tests`` Django project::
 
 Run docker::
 
-    docker-compose up
+    docker compose up
```

Comment:
The macOS command is still `docker-compose`.

Reply:
Does `docker compose` work as well?